packages/default-theme/src/logic/useLocales.js

```diff
@@ -0,0 +1,36 @@
+import { computed } from "@vue/composition-api"
```

Comment:
Do you leave it for compatibility purposes?

Reply:
yes, I marked it as deprecated as well...